vue-components/src/components/Popover.vue

```diff
@@ -264,6 +440,19 @@ $pointer-edge-length: math.hypot($wikit-Popover-pointer-width/2, $wikit-Popover-
 			}
 		}
 
+		&#{$base}--flush-edges #{$base}__content-wrapper {
+
+			@if $subposition == start {
+				inset-inline-end: min(50% - #{$wikit-Popover-pointer-width} / 2 -
+				#{$wikit-Popover-pointer-margin-horizontal}, 0px);
```

Comment:
I'm still struggling to fully wrap my head around this. Why are we doing this calculation, instead of just setting it 0px directly?

Reply:
You can’t see it in the storybook (which is maybe a shortcoming that should be addressed anyways), but on very narrow buttons (such as icon buttons), this length becomes negative, and that’s necessary to ensure that the pointer is centered over the button.

With original `calc`:
![Screenshot 2021-06-16 at 13-08-48  wmde query-builder](https://user-images.githubusercontent.com/2346599/122208786-0b088c00-cea4-11eb-8b51-7b21d594ca25.png)
With `0`:
![Screenshot 2021-06-16 at 13-08-14  wmde query-builder](https://user-images.githubusercontent.com/2346599/122208788-0c39b900-cea4-11eb-8801-8d8123e36038.png)

Reply:
ah, gotcha, thanks